environment.py

```diff
@@ -216,7 +228,10 @@ def read_metric_values(self):
                         intervalId=inv_obj.INSTANT_INTERVAL,
                         maxSample=1, format='normal'
                     )
-                    results = perf_manager.QueryPerf(querySpec=[query_spec])
+                    try:
+                        results = perf_manager.QueryPerf(querySpec=[query_spec])
+                    except Exception as e:
+                        self._logger.error("Exception while making performance query : {0}".format(e))
```

Comment:
Similar issue here I think. If there's an exception raised `results` won't be set. Should it return from the function here?